startReactCommerce.sh

```diff
@@ -61,6 +61,8 @@ token=$(cat ~/.npmrc |  grep -o '\/\/verdaccio\.ubreakit\.com\/:_authToken="[^"]
 if [ -z "$token" ]; then
     echo "Token not found in ~/.npmrc"
     return 1
+else
+    export VERDACCIO_AUTH_TOKEN=$token
```

Comment:
Does this work on windows machines? I believe windows uses `set` instead of `export`.

Reply:
I added a check for windows machines, but please let me know if that's not a good approach